go-controller/pkg/ovn/ovn.go

```diff
@@ -544,6 +552,30 @@ func (oc *Controller) WatchNetworkPolicy() error {
 	return err
 }
 
+// WatchEgressFirewall starts the watching of egressfirewall resource and calls
+// back the appropriate handler logic
+func (oc *Controller) WatchEgressFirewall() error {
+	_, err := oc.watchFactory.AddEgressFirewallHandler(cache.ResourceEventHandlerFuncs{
+		AddFunc: func(obj interface{}) {
+			egressFirewall := obj.(*egressfirewall.EgressFirewall)
+			klog.Errorf("KEYWORD: ADDING NEW EGRESSFIREWALL")
```

Comment:
is this intended? looks like some debugging statement

Reply:
not intended, I deleted the old debug line